sunpy/map/header_helper.py

```diff
@@ -24,7 +28,7 @@ def meta_keywords():
          'cunit2': 'Units of the coordinate increments along naxis2 e.g. arcsec **required',
          ...
     """
-    return _map_meta_keywords
+    return MappingProxyType(_map_meta_keywords)
```

Comment:
I think this counts as an API breaking change which we can't do. 
Can this not return a copy of _map_meta_keywords each time?